### PR TITLE
Add Installation instructions

### PR DESCRIPTION
**Pull request by limonte**
Connected to #148

After the npm package has been released, it makes sense to inform users about it.

### CODE REVIEW

**Comment by limonte** [comment]
> Could you please bump the package version to 0.01.02 

First, that would be totally out of the scope of this PR. 
Secondly, why `x.x.02`? It should be `x.x.01` as the current patch version is `0`.
Third, according to `semver`, it should be without leading zeros - `0.1.1`. 

> include some more detail about how/why to use the installed version.

Well, I can add the answer on question `how?`, but not `why?` as the latest seems to be well-explained in the README already.

**Comment by edent** [comment]
Ah, sorry, I hadn't pushed my local changes. I've already published 0.1.2 (you are correct about semver).

If you can add some more about the how - that'd be great.

**Comment by limonte** [comment]
Will do :+1:

File: README.md [hunk 1]
```diff
@@ -5,9 +5,33 @@ The logos have a 512x512 viewbox, they will fit in a circle with radius 256. The
 
 Say thanks!
 
-* <a href="https://amzn.to/2o95rA8"><img src="https://edent.github.io/SuperTinyIcons/images/svg/amazon.svg" width="40" /> Gift me something from my Amazon Wishlist
-* <a href="https://paypal.me/edent/gbp5"><img src="https://edent.github.io/SuperTinyIcons/images/svg/paypal.svg" width="40" /> Send me a tip
-* <a href="https://edent.tel/"><img src="https://edent.github.io/SuperTinyIcons/images/svg/whatsapp.svg" width="40" /> Leave me a message
+* <a href="https://amzn.to/2o95rA8"><img src="https://edent.github.io/SuperTinyIcons/images/svg/amazon.svg" width="40" /> Gift me something from my Amazon Wishlist</a>
```
Comment (limonte) [inline README.md:8]: closing `</a>` were missing here

File: README.md [hunk 2]
```diff
@@ -5,9 +5,33 @@ The logos have a 512x512 viewbox, they will fit in a circle with radius 256. The
...
+
+The modern way, the React (JSX) example:
+
+```jsx
+import logo from 'super-tiny-icons/images/svg/github.svg';
+
+<img src={logo} />
+```
+
+The demo repository with bootstrapped with `create-react-app`: [create-react-app-super-tiny-icons](https://github.com/limonte/create-react-app-super-tiny-icons)
```
Comment (limonte) [inline README.md]: feel free to fork the demo repo to your account and update the link

**Comment by limonte** [comment]
> If you can add some more about the how - that'd be great.

Done, take a look :+1:

**Pull request merged**